tests/ensure_non_running_provider.py

```diff
@@ -21,6 +21,8 @@
     "tests_vlan_mtu_nm.yml",
```

Comment:
Please adjust the commit message to follow https://chris.beams.io/posts/git-commit/